pkgs/development/tools/phase/update.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq nix-prefetch common-updater-scripts nix coreutils
+# shellcheck shell=bash
+set -euo pipefail
+
+# Fetch the latest Phase CLI version from GitHub https://github.com/phasehq/cli
+REPO="phasehq/cli"
+RELEASE_TAG=$(curl -s "https://api.github.com/repos/$REPO/releases/latest" | jq -r .tag_name)
+VERSION=${RELEASE_TAG#v}
+
+echo "Latest Phase CLI release: $VERSION"
+
+# Prepare x86 and arm64 targets for Linux and MacOS
+ARCHS=(
+    "x86_64-linux:linux_amd64"
+    "x86_64-darwin:macos_amd64"
+    "aarch64-linux:linux_arm64"
+    "aarch64-darwin:macos_arm64"
+)
+
+# Paths to default.nix and SRI (Sub-resource integrity) file
+NIXCONFIG=pkgs/development/tools/phase/default.nix
+SRIFILE=pkgs/development/tools/phase/hashes.json
+
+# Clear file from a previous run
+rm -f "$SRIFILE" && touch "$SRIFILE"
+
+printf "{ \"_comment\": \"@generated by pkgs/development/tools/phase/update.sh\"\n" >> "$SRIFILE"
```

Comment:
Pretty sure you can just directly override a file if you use a write pipe instead of an append one. But still, it would be a better idea to keep the old hash file around, and restore it if the update script fails in any way.

Reply:
Yeah, was thinking the same thing. Fixed.